parm/config/gfs/config.aeroanlgenb

```diff
@@ -10,8 +10,9 @@ source "${EXPDIR}/config.resources" aeroanlgenb
 
 export BMATYAML="${PARMgfs}/gdas/aero/berror/aero_diagb.yaml.j2"
 export DIFFUSIONYAML="${PARMgfs}/gdas/aero/berror/aero_diffusionparm.yaml.j2"
-export iterations=10
-export fixed value=1.0
+export diff_iter=10
```

Comment:
@andytangborn we should name these something like `aero_var_diff_iter` just so it is more explicit

Reply:
> @andytangborn we should name these something like `aero_var_diff_iter` just so it is more explicit

Sure.

Reply:
So these will probably also need changed in GDASApp templates?

Reply:
> > @andytangborn we should name these something like `aero_var_diff_iter` just so it is more explicit
> 
> Sure.

I'm assuming this only needs to change in config.aeroanlgenb and aero_diffusionparm.yaml.j2   

also, I think "aero_diff_iter" makes the most sense, because this isn't in the "var" step, but rather the diffusion parameters step.

Reply:
yeah I think that's all the places where it needs changed.

Good point about the var, what about `"aero_b_diff_iter"`? or `"aero_diffusion_iter"` just to make sure it's not read by someone as a difference of aerosols or something else from what ti actually is

Reply:
> yeah I think that's all the places where it needs changed.
> 
> Good point about the var, what about `"aero_b_diff_iter"`? or `"aero_diffusion_iter"` just to make sure it's not read by someone as a difference of aerosols or something else from what ti actually is

Yes these seem fine.  I can do the changes in GDASApp first.